src/engine/controls/loopingcontrol.cpp

```diff
@@ -1495,6 +1509,14 @@ void LoopingControl::slotBeatJump(double beats) {
     }
 }
 
+void LoopingControl::slotBeatJumpSizeChangeRequest(double beats) {
+    if ((beats < kMinBeatJumpSize) || (beats > kMaxBeatJumpSize)) {
+        // Don't clamp the value here to not fall out of a measure
```

Comment:
since valid beatloop and beatjump sizes are identical, let's use
```
double maxBeatSize = s_dBeatSizes[sizeof(s_dBeatSizes) / sizeof(s_dBeatSizes[0]) - 1];
double minBeatSize = s_dBeatSizes[0];
```
like in `slotBeatLoopSizeChangeRequest` above and omit the constants.